_rules/remap-shortcut-aa8b52.md

```diff
@@ -0,0 +1,337 @@
+---
+id: aa8b52
+name: Keyboard shortcut can be remapped
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, a mechanism to remap the shortcut to use one or more non-printable characters exists.
+accessibility_requirements: # Remove whatever is not applicable

```

Comment:
I think adding accessibility requirement mapping to specific technique can be a good idea. See #832.

Reply:
Hmm, forget it this rule is actually not mapping to G127 because G127 is "remap or disable", so "no remap" does no fail G127…